src/com/jalasoft/search_files/search/search_criteria.py

```diff
@@ -21,8 +21,9 @@ def __init__(self, criteria=3, path='/'):
             "criteria": criteria,
             "path": path,
             "size": None,
-            "init_date": None,
-            "end_date": None,
+            "creation_date": {"start_date": None, "end_date": None},
+            "modification_date": {"start_date": None, "end_date": None},
+            "last_access_date": {"start_date": None, "end_date": None},
```

Comment:
do you think this could be used in a single one something like this
filter_for_date = {"filter_type": "*****", "start_date" = None, "end_date" = None}
where ***** would be if it's creation or modification, seams to me a lot of unnecessary dictionaries for the same variables

Reply:
what about if in some case user wants to add all the dates to filter their search